packages/node_modules/@node-red/runtime/lib/storage/localfilesystem/projects/index.js

```diff
@@ -499,6 +504,7 @@ function getFlows() {
     if (!initialFlowLoadComplete) {
         initialFlowLoadComplete = true;
         log.info(log._("storage.localfilesystem.user-dir",{path:settings.userDir}));
+        log.info(log._("storage.localfilesystem.projects.projects-folder", {projectsFolder: projectsDir}));
```

Comment:
We should only log this if the projects feature is enabled.
```suggestion
        if (projectsEnabled) {
            log.info(log._("storage.localfilesystem.projects.projects-folder", {projectsFolder: projectsDir}));
        }
```